src/Mod/Sketcher/Gui/ViewProviderSketch.cpp

```diff
@@ -3237,6 +3244,12 @@ QString ViewProviderSketch::getPresentationString(const Constraint *constraint)
         userStr.insert(0, QChar(82)); // Capital letter R
     }
 
+    if (showNameParam && !nameStr.isEmpty())
+    {
+        userStr.insert(0, QString::fromLatin1(": "));
+        userStr.insert(0, nameStr);

```

Comment:
QString has a good concatenation system, something like that should work : 
```suggestion
        userStr = nameStr + " :" + userStr
```

Reply:
Better.